src/features/cookies/helpers/removeGenerateCookieKey.test.ts

```diff
@@ -0,0 +1,38 @@
+import { removeGenerateCookieKey } from 'features/cookies/helpers/removeGenerateCookieKey'
+import { storage, StorageKey } from 'libs/storage'
+import { renderHook, superFlushWithAct } from 'tests/utils'
+
+const key = 'traffic_medium'
+const keyPrefix = 'algoliasearch-client-js' as StorageKey
+const keyWithPrefix = keyPrefix.concat('-', '1234') as StorageKey // algoliasearch-client-js-1234
+const keyWithPartialPrefix = keyPrefix.slice(0, keyPrefix.length / 2) as StorageKey // algoliasear
+const someValue = 'some value'
+
+describe('removeGenerateCookieKey', () => {
+  it('should clear optional key', async () => {
+    storage.saveObject(key, someValue)
+
+    renderHook(() => removeGenerateCookieKey(key))
+
+    await superFlushWithAct()
```

Comment:
Tu n'as pas besoin de `act` donc tu peux utiliser `await flushAllPromises()`
```suggestion
    await flushAllPromises()
```